### PR TITLE
feat: add finch version output to support-bundle

### PR DESCRIPTION
**Pull request by swagatbora90**
Dumps finch version output to a file when generating finch support-bundle

*Testing done:*

`make test-unit` and `make test-e2e`

- [x ] I've reviewed the guidance in CONTRIBUTING.md

#### License Acceptance

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pkg/support/support.go [hunk 1]
```diff
@@ -320,6 +328,16 @@ func getFinchVersion() string {
 	return version.Version
 }
 
+func (bb *bundleBuilder) getFinchVersion() string {
+	cmd := bb.ecc.Create("finch", "version")
```
Comment (pendo324) [inline pkg/support/support.go:332]: is it worth replacing "finch" with `os.Executable()`?

**Review by pendo324 — COMMENTED** [review]
One question, otherwise LGTM pending the linter fixes

File: pkg/support/support.go (hunk 1 above)
Comment (swagatbora90) [reply pkg/support/support.go:332]: Sure, I can change this. I have to fix a few lint failures as well. Sending a new revision with all the changes.

Comment (pendo324) [reply pkg/support/support.go:332]: Cool. You can use the `ExecutableFinder` in `pkg/system` to make mocking easier. There's some other usages in the code base

**Review by pendo324 — APPROVED** [review]

**Review by Kern-- — APPROVED** [review]

**Pull request merged**